es5/README.md

```diff
@@ -429,7 +429,7 @@
 
     ```javascript
     // bad
-    function() {
+    function () {
```

Comment:
I actually prefer the space, personally.  It reminds you that there's a space where you could put a function name, which I find to be useful whenever defining longer functions.  It can be useful when trying to debug a stracktrace for example.